docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Device Identity
+description: >-
+  Learn about how WebAuthn is used to authenticate and authorize access using Device ID and state.
+sidebarDepth: 1
+---
+
+# Device Identity
+
+One of the core components of a zero-trust security model device identity, whereby a device's identity and state can be authenticated and factored into access control decisions. This topic page covers the concept of device identity and how it applies to the zero-trust model.
```

Comment:
Yea, it was a bad copy/pasta of parts from previous iterations. How about:
```suggestion
One of the core components of a zero-trust security model is **device identity**, which is the ability for a device to have a unique, unclonable identity string that can be authenticated and factored into access control decisions. This topic page covers the concept of device identity, and how it applies to the zero-trust model.
```

I didn't include your changes because we say "a zero-trust security model" at the start of the paragraph, and I don't want it to read as redundant.